apps/address-service/domains/common/utils/services/suggest/providers/DadataSuggestionProvider.js

```diff
@@ -256,7 +258,7 @@ class DadataSuggestionProvider extends AbstractSuggestionProvider {
     /**
      * @returns {Promise<DadataObject[]>}
      */
-    async get ({ query, context = null, count = 20, helpers = {} }) {
+    async get ({ query, context = '', count = 20, helpers = {} }) {
```

Comment:
Seems like context is not a string-type :)

Reply:
I changed the type to non nullable string